include/swift/ABI/Metadata.h

```diff
@@ -301,16 +302,19 @@ struct ValueWitnessTable {
   value_witness_types::LOWER_ID LOWER_ID;
```

Comment:
We'll also need to targetize this part. For the `InProcess` target these should be function pointers, otherwise they ought to be `StoredPointer`s of the target pointer size.

Reply:
Oh yes, I missed these, thanks. It's not obvious to me how to generalize these, the best thing I was able to come up with is creating a new namespace `target_value_witness_types` with the target dependent types to be used in `TargetValueWitnessTable`. Would that make sense to you?